script/distanceOfErrMsg.pl

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/perl
+

```

Comment:
I would really recommend to enable strict/warnings when writing Perl code. Since you'll likely have Mojolicious present anyways, just put `use Mojo::Base -strict, -signatures;` at the beginning of the script.